### PR TITLE
Add consistent testing to actions [SyncWith: zkLLVM#289]

### PR DESCRIPTION
**Pull request by x-mass**

### CODE REVIEW

**Review by martun — APPROVED** [review]

File: .github/workflows/build_linux.yml [hunk 1]
```diff
@@ -14,9 +14,20 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  handle-syncwith:
+    if: github.event_name == 'pull_request'
+    name: Call Reusable SyncWith Handler
+    uses: NilFoundation/ci-cd/.github/workflows/reusable-handle-syncwith.yml@0588663776c5989c2ba23064811e7f71a20ada83
+    with:
+      ci-cd-ref: '0588663776c5989c2ba23064811e7f71a20ada83'
```
Comment (martun) [inline .github/workflows/build_linux.yml:22]: Do we need this commit hashes here?

**Review by martun — APPROVED** [review]

File: .github/workflows/build_linux.yml (hunk 1 above)
Comment (x-mass) [reply .github/workflows/build_linux.yml:22]: This way we won't brake current CI even with bad commit merged into ci-cd repo. If we will frequently modify this, we should better replace it with branch and make that branch protected

Comment (nkaskov) [reply .github/workflows/build_linux.yml:22]: Maybe we could use a tag instead?

**Review by nkaskov — APPROVED** [review]

**Review by makxenov — APPROVED** [review]

File: .github/workflows/build_linux.yml (hunk 1 above)
Comment (x-mass) [reply .github/workflows/build_linux.yml]: I replaced SHAs with tags to make it more pleasing to the eye, but we will have to update it manually anyway

**Pull request merged**